packages/blocks/src/lib/navigation-map/components/axes-helper.svelte

```diff
@@ -29,7 +29,7 @@ export let length = 10;
   scale.z={thickness}
   position.y={length / 2}
 >
-  <T is={new THREE.MeshBasicMaterial({ color: 'green' })} />
+  <T is={new THREE.MeshBasicMaterial({ color: 'blue' })} />
```

Comment:
These were incorrect. x,y,z should correspond to r,g,b, so I quickly fixed it up!